CMakeLists.txt

```diff
@@ -93,6 +93,7 @@ option (WITH_ALEMBIC                        "Build Alembic support"
 option (WITH_OSL                            "Build OSL support"                                     OFF)
 option (WITH_DISNEY_MATERIAL                "Build Disney material"                                 OFF)
 option (WITH_NORMALIZED_DIFFUSION_BSSRDF    "Build Normalized diffusion BSSRDF (patented)"          OFF)
+option (WITH_DIRECTIONAL_DIPOLE_BSSRDF      "Build Directional dipole BSSRDF"                       OFF)
```

Comment:
I agree. If we can fix it, we should remove the option. In the meantime, I think it's ok to have the option to disable it.